catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -168,6 +168,15 @@ protected JdbcTable.Builder getTableBuilder(
     return builder;
   }
 
+  protected JdbcColumn.Builder getColumnBuilder(
```

Comment:
How about adding the `TABLE_SCHEM` check here, and remove the `getColumnBuilder ` in PG?